lib/src/form_builder_validators.dart

```diff
@@ -172,6 +172,48 @@ class FormBuilderValidators {
     };
   }
 
+  /// [FormFieldValidator] that requires the words count of the field's value to be
+  /// greater than or equal to the provided minimum count.
+  static FormFieldValidator<String> minWordsCount(
+    int minCount, {
+    bool allowEmpty = false,
+    String? errorText,
+  }) {
+    assert(minCount > 0);
```

Comment:
Please add a assert detail message to explain better why got the assert and how solve
https://dart.dev/guides/language/language-tour#assert

Reply:
Added ✅